release/_bom.sh

```diff
@@ -273,7 +273,7 @@ function generate_pom_release_bom {
 				group_id="com.liferay"
 			fi
 
-			if (grep -q "<artifactId>${artifact_id}</artifactId>" "${pom_file_name}" && grep -q "<groupId>${group_id}</groupId>" "${pom_file_name}")
+			if (grep -q "\t\t\t\t<groupId>${group_id}</groupId>\n\t\t\t\t<artifactId>${artifact_id}</artifactId>\n\t\t\t\t<version>${version}</version>" "${pom_file_name}")
```

Comment:
@lucasmiranda0, can you please replace the existing `\t` by a regex, like `(\t)*`?